odoo/tools/misc.py

```diff
@@ -802,6 +803,45 @@ def deco(*args, **kwargs):
                 return func(*args, **kwargs)
         return deco
 
+    def emit(self, record):
+        pass
+
+
+class quiet_logging(logging.Handler):
```

Comment:
What about `lower_logging` instead, since it is what it does?
Also, I would pass the maximal logging level as a parameter, instead of hardcoding `25`.